### PR TITLE
Lore module

### PR DESCRIPTION
**Pull request by lordabrax**

### CODE REVIEW

File: app/indexer/LoreIndexer.ts [hunk 1]
```diff
@@ -1,86 +1,51 @@
...
-  private CONTRACTS = [
-    "0x521151a3e28d2efff7d7d34c4f8bbe45c4c9f37424bae4dc63836fd659c51e6"
-  ];
-  // @ts-ignore
-  private context: Context;
-  private contract: Contract = new Contract(LoreABI as any, this.CONTRACTS[0]);
+const CONTRACT = "0x06894a6766b4763d8bea8d43f433d25e577ed8bf057942c861df4e9951282c64";
 
+export default class LoreIndexer extends BaseContractIndexer {
+  private contract: Contract = new Contract(LoreABI as any, CONTRACT);
```
Comment (tenoxious) [inline app/indexer/LoreIndexer.ts]: Do we need to pass in Provider here?
e.g.
```
 this.contract = new Contract(LoreABI as any, CONTRACT, new Provider({
  network: NETWORK === "goerli" ? "goerli-alpha" : "mainnet-alpha"
}));

File: prisma/schema.prisma [hunk 2]
```diff
@@ -255,8 +258,14 @@ model LoreEntityRevision {
   entity   LoreEntity @relation(fields: [entityId], references: [id])
   entityId Int
 
-  pois  LorePoisOnEntityRevisions[]
-  props LorePropsOnEntityRevisions[]
+  // For Starknet indexing
+  eventIndexed      String?
+
+  // For Starknet indexing
+  eventIndexed      String?
```
Comment (tenoxious) [inline prisma/schema.prisma]: duplicate eventIndexed
giving error

Comment (lordabrax) [reply prisma/schema.prisma]: Oh, didn't notice while pulling. Thanks, will fix!

File: app/indexer/LoreIndexer.ts (hunk 1 above)
Comment (lordabrax) [reply app/indexer/LoreIndexer.ts]: Good idea! Just added.

**Review by tenoxious — APPROVED** [review]
🎉

**Pull request merged**